examples/asr/conf/fastconformer/fast-conformer-long_ctc_bpe.yaml

```diff
@@ -0,0 +1,203 @@
+# It contains the default values for training a Fast Conformer-CTC ASR model, large size (~120M) with CTC loss and sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
```

Comment:
Have you trained/evaluated with this exact configuration?

Reply:
Yes, it gets slightly worse results on ls than baseline